dumper/app/src/main/resources/oracle-stats/native/db-objects-no-join.sql

```diff
@@ -0,0 +1,63 @@
+-- Copyright 2022-2024 Google LLC
```

Comment:
I would simply name this script as `db-objects`.

Reply:
Done